plugins/feeds/public/botvrij_hostname.py

```diff
@@ -1,38 +1,38 @@
 import logging
 from datetime import timedelta, datetime
-from core.errors import ObservableValidationError
-from core.feed import Feed
-from core.observables import Hostname
+from core.schemas import observable
+from core.schemas import task
+from core import taskmanager
 
 
-class BotvrijHostname(Feed):
-    default_values = {
+class BotvrijHostname(task.FeedTask):
```

Comment:
Consider grouping all the botvrij_* feeds into the same file?

Reply:
The architecture is a file by source. If we change that, we have to do the same thing for data plane. And after we want to Factorising the code 😬